community/coreutils/build

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+export CC="gcc -static"
+
+export FORCE_UNSAFE_CONFIGURE=1
```

Comment:
Why is this needed?

Reply:
It is my opinion static linking with musl is the way to go as much as possible. Is this not the KISS position?

For example stal.li linux has some short write-ups on the subject re: the benefits of static...

> Executing statically linked executables is much faster, because there are no expensive shared object lookups during exec().
> Statically linked executables are portable, long lasting and fail safe to ABI changes -- they will run on the same architecture even in 10 years time. Never expect errors like /lib/ssa/libstdc++.so.6:version 'GLIBCXX_3.4.4' not found again.
> Statically linked executables use less disk space. Most executables use only a small subset of the functions provided by a static library -- so there is absolutely no reason to link complete static libraries into a static executable (e.g. spoken for a hello_world.c you only need to link vprintf statically into the executable, not the whole static libc!). The contrary is true for dynamic libraries -- you always use the whole library, regardless what functions you are using.
> Statically linked executables consume less memory because their binary size is smaller and they only map the functions they depend on into memory (contrary to dynamic libs).
http://web.archive.org/web/20090525150626/http://blog.garbe.us/2008/02/08/01_Static_linking/

Also see https://sta.li/faq/

Reply:
@dylanaraps or does your question refer to the line:
`export FORCE_UNSAFE_CONFIGURE=1`
?

Reply:
export FORCE_UNSAFE_CONFIGURE=1

is required by configure because we are building as the root user in KISS and gnu does not feel that doing that is a good idea.

Reply:
In KISS the package manager should be run as your normal user actually. Only the install/remove package manager operations need root. 

Yeah, I was referring to `FORCE_UNSAFE_CONFIGURE=1`.

Reply:
Hmmm. I much prefer running all of kiss commands as root and not typing my su password all the time ... :( Can we not keep this environment variable?

Reply:
If you install `sudo` it will only prompt once for the password as `sudo` saves your auth for a period of time by default. I'm fine with keeping it around for your use case though.

Reply:
thanks, please keep it around. I'm anti-sudo. lol